### PR TITLE
Write data for drydep_inti_xactive

### PR DESCRIPTION
**Pull request by brigaudet**
Write out input / output data fields for drydep_inti_xactive subroutine in mo_drydep.F90.
Also include code to write out integer 3D variables in yaml / python files.

### CODE REVIEW

File: components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_input_ts_0.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
...027763E+001,  0.11647151510034477E+001,  0.12617568795285550E+001,  0.12617568795285550E+001,  0.14321972701768837E+001,  0.12617568795285550E+001,  0.11647151510034477E+001,  0.14321972701768837E+001,  0.12617568795285550E+001,  0.10180517035673147E+001,  0.85017381732928443E+000,  0.10724153591027763E+001,  0.87952191739420205E+000,  0.87952191739420216E+000,  0.10724153591027763E+001,  0.85017381732928432E+000,  0.10180517035673147E+001,  0.12617568795285550E+001,  0.14321972701768837E+001,  0.11647151510034477E+001,  0.12617568795285550E+001,  0.14321972701768837E+001,  0.12617568795285550E+001,  0.12617568795285550E+001,  0.11647151510034477E+001,  0.10724153591027763E+001,  0.87952191739420205E+000,  0.10180517035673147E+001,  0.85017381732928443E+000,  0.79129952552206684E+000,  0.91987191184397255E+000,  0.70497830836867026E+000,  0.79129952552206673E+000,  0.10180517035673147E+001,  0.10724153591027763E+001,  0.85017381732928443E+000,  0.87952191739420205E+000,  0.10724153591027763E+001,  0.10180517035673147E+001,  0.87952191739420205E+000,  0.85017381732928443E+000,  0.91987191184397255E+000,  0.79129952552206684E+000,  0.79129952552206684E+000,  0.70497830836867037E+000]
```
Comment (odiazib) [inline components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_input_ts_0.yaml:6]: We are missing the header of this yaml file:

```
mam4xx:
  function: drydep...
input:
```

File: components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_output_ts_0.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...763E+001,  0.11647151510034477E+001,  0.12617568795285550E+001,  0.12617568795285550E+001,  0.14321972701768837E+001,  0.12617568795285550E+001,  0.11647151510034477E+001,  0.14321972701768837E+001,  0.12617568795285550E+001,  0.10180517035673147E+001,  0.85017381732928443E+000,  0.10724153591027763E+001,  0.87952191739420205E+000,  0.87952191739420216E+000,  0.10724153591027763E+001,  0.85017381732928432E+000,  0.10180517035673147E+001,  0.12617568795285550E+001,  0.14321972701768837E+001,  0.11647151510034477E+001,  0.12617568795285550E+001,  0.14321972701768837E+001,  0.12617568795285550E+001,  0.12617568795285550E+001,  0.11647151510034477E+001,  0.10724153591027763E+001,  0.87952191739420205E+000,  0.10180517035673147E+001,  0.85017381732928443E+000,  0.79129952552206684E+000,  0.91987191184397255E+000,  0.70497830836867026E+000,  0.79129952552206673E+000,  0.10180517035673147E+001,  0.10724153591027763E+001,  0.85017381732928443E+000,  0.87952191739420205E+000,  0.10724153591027763E+001,  0.10180517035673147E+001,  0.87952191739420205E+000,  0.85017381732928443E+000,  0.91987191184397255E+000,  0.79129952552206684E+000,  0.79129952552206684E+000,  0.70497830836867037E+000,],]
```
Comment (odiazib) [inline components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_output_ts_0.py]: We are missing this part of the code:
```
from math import nan as nan, inf as inf

# Object is just a dynamic container that stores input/output data.
class Object(object):
    pass
# Settings are stored here.
settings = Object()
# Input is stored here.
input = Object()
```

**Review by odiazib — CHANGES_REQUESTED** [review]

File: components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_input_ts_0.yaml (hunk 1 above)
Comment (odiazib) [reply components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_input_ts_0.yaml:6]: as function name, could you use `find_season_index`?

Comment (brigaudet) [reply components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_input_ts_0.yaml:6]: This should be fixed now.

File: components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_output_ts_0.py (hunk 2 above)
Comment (brigaudet) [reply components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_output_ts_0.py]: This should be fixed.

**Review by odiazib — APPROVED** [review]

**Comment by brigaudet** [comment]
image
All tests passed will merge.

**Pull request merged**